package.json

```diff
@@ -5,6 +5,7 @@
 		"build:deploy-preview": "npx --package @hashicorp/platform-tools@0.5.0 hc-tools --resolve-paths false ./scripts/content-repo-preview/build.ts",
 		"build:webpack-only": "hc-tools next-build-webpack-only",
 		"build": "next build",
+		"clean": "rimraf .next node_modules",
```

Comment:
💭 You should be able to just use `rm -rf` I think? [See reference here ](https://github.com/hashicorp/learn/blob/master/package.json#L153). Doesn't matter a ton though 😸

Reply:
Oh I see your note in the description on why you used `rimraf`, feel free to disregard comment above.